### DISCUSSION TITLE
Academic Discount Issue for Primary Email Address

### DISCUSSION THREAD

**Discussion by kanglimim** [community/community#145005] [GitHub Education]
### Select Topic Area

Question

### Body

Hello GitHub Support Team,

Thank you for your response. However, I believe there has been some misunderstanding regarding my issue.

The email address  set as the primary email for my GitHub account, and I am trying to apply for GitHub Education benefits. It seems that this email address was previously associated with another account (kanglimlim).

As this email address is now my primary email and I use it for my main GitHub account, I kindly request assistance in resolving this issue so that I can successfully apply for the Education benefits.

If additional verification is required to confirm ownership or identity, please let me know.

Thank you for your understanding and support.

Best regards,

**Comment by shinybrightstar** [comment]
Welcome to the GitHub Community, @kanglimim, we're happy you're here! You are more likely to get a useful response if you are posting your question(s) in the applicable category and are explicit about what your project entails--giving a few more details might help someone give you a nudge in the right direction. I've gone ahead and moved it to Github Education for you. Good luck!
